src/explore-education-statistics-common/src/modules/table-tool/components/MultiHeaderTable.module.scss

```diff
@@ -47,34 +47,39 @@ $border-width: 1px;
 
 .tableHead {
   td {
-    background: #fff;
+    background: govuk-colour('white');
     position: relative;
     z-index: 3;
   }
 
-  .emptyColumnHeaderCell {
-    background: govuk-colour('light-grey');
-    height: 46px;
-    z-index: 1;
+  // Uses the spacer cell to cover the left border
+  // of header cells. We can't just remove the
+  // left border on the first cell in header rows
+  // because this causes inconsistent styling when there
+  // are merged cells which span down through rows.
+  td::after {
+    background: govuk-colour('white');
+    bottom: 0;
+    content: '';
+    position: absolute;
+    right: -1px;
+    top: 0;
+    width: 1px;
   }
 
   th {
     background: govuk-colour('light-grey');
     border-bottom: 1px solid $govuk-border-colour;
-    border-right: 1px solid $govuk-border-colour;
+    border-left: 1px solid $govuk-border-colour;
```

Comment:
Super minor comment but I'm just wondering if we perhaps want a snapshot test for this in order to be an early warning system when someone might end up changing these styles in the future?

Reply:
A snapshot test won't work here unless we're talking about visual regression. That's a lot more work than just this ticket though.